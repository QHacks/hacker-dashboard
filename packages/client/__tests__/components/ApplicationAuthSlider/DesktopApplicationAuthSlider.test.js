import React from "react";
import DesktopApplicationAuthSlider from "../../../components/ApplicationAuthSlider/DesktopApplicationAuthSlider";
import { shallow } from "enzyme";

describe("ApplicationAuthSlider", () => {
  it("renders correctly", () => {
    expect(
      shallow(<DesktopApplicationAuthSlider items={["TEST"]} />)
    ).toMatchSnapshot();
  });
});
