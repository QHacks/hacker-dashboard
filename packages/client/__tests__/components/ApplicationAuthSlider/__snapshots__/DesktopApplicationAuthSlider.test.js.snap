// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ApplicationAuthSlider renders correctly 1`] = `
<div
  className="css-s9fkrg"
>
  <button
    className="item"
    key="TEST"
    onClick={[Function]}
    type="button"
  >
    TEST
  </button>
  <div
    className="css-1do9vzn"
    role="none presentation"
  />
</div>
`;
